docs/src/main/sphinx/release/release-378.md

```diff
@@ -0,0 +1,59 @@
+# Release 378 (20 Apr 2022)
+
+## General
+
+* Add {func}`to_base32` and {func}`from_base32` varbinary functions. ({issue}`11439`)
+* Improve planning performance of queries with large `IN` lists.
+  ({issue}`11902`, {issue}`11918`, {issue}`11956`)
+* Fix reporting of total spilled bytes in JMX metrics. ({issue}`11983`)
+
+## Security
+
+* Require value for the ``internal-communication.shared-secret`` configuration
+  property when authentication is enabled. ({issue}`11944`)
+
+## CLI
+
+* Allow disabling progress reporting during query executing in the CLI client by
+  specifying `--no-progress` ({issue}`11894`)
+* Avoid adding about 200ms of delay to very short queries. ({issue}`11768`)
+
+## Delta Lake connector
+
+* Improve query planning performance. ({issue}`11858`)
+* Fix reading from `information_schema.columns` when metastore contains views. ({issue}`11946`)
+
+## Hive connector
+
+* Improve query planning performance. ({issue}`11858`)
+
+## Iceberg connector
+
+* Add support for hidden `$path` columns. ({issue}`8769`)
+* Add support for creating tables with either Iceberg format version 1, or 2. ({issue}`11880`)
+* Allow reading Iceberg tables written by Glue that have locations containing
+  double slashes. ({issue}`11964`)
+* Improve query planning performance. ({issue}`11858`)
+
+## MySQL connector
+
+* Improve query planning performance. ({issue}`11858`)
+
+## Pinot connector
+
+* Support querying tables having non-lowercase names in Pinot. ({issue}`6789`)
```

Comment:
`non-lowercase` -> `upper case`?

Reply:
also mixed case and such...  so I think it has to stay as is